ui/app/AppLayouts/Chat/ContactsColumn/Channel.qml

```diff
@@ -165,12 +165,12 @@ Rectangle {
           wrapper.hovered = false
         }
         onClicked: {
+            chatsModel.setActiveChannelByIndex(index)
+            chatGroupsListView.currentIndex = index
```

Comment:
After a bit of thought, I decided I need to redo this change here.
Please don't mind this change.